dali/operators/reader/video_reader_op.h

```diff
@@ -161,6 +160,9 @@ class VideoReader : public DataReader<GPUBackend, SequenceWrapper> {
 
     ProcessVideo(video_output, curent_batch, ws);
 
+    // set layout here as ProcessVideo may have changed it via Copy or other function
+    video_output.SetLayout("FHWC");
```

Comment:
I think Copy should copy the layout as well

Reply:
Synced offline. Done.